### ISSUE TITLE
Enable Germinate Datasource, without requiring url option

### ISSUE THREAD

**Issue by Don-Isdale** [plantinformatics/pretzel#375]
2024May17
Germinate, on plantinformatics.io : no feature flag required : should be on by default.

---
2024May20
from chat (Pretzel development team weekly), 4:04 PM : G :
- append (Experimental) to some names, e.g. Germinate, Spark, BrAPI

**Comment by Don-Isdale** [comment]
#### Implementation

- [x] [feature/ongoingGenotype d893a7d2] closes #375 Enable Germinate Datasource, without requiring url option

#### Testing

Test :
-  open Pretzel in a new tab
- click to add a Datasource - the New Datasource modal dialog is displayed
- click the Server Type pull-down, and it displays 2 options : Pretzel and Germinate.  This is shown in the following screen-shot.

![Screenshot from 2024-07-08 15-27-30](https://github.com/plantinformatics/pretzel/assets/2305340/cca173a8-ce6a-4758-8abe-6e9a824661d8)

**Comment by gabrielkg** [comment]
Merged and available in feature/upgradeFrontend2. To be tested once #370 is merged into master.
